### PR TITLE
webpack configuration fixes

### PR DESCRIPTION
**Pull request by FunkyColdMedyna**
minor web-pack config alterations. 

Resolved some minor error messages from previous commits

### CODE REVIEW

**Review by hjkmines — APPROVED** [review]
lgtm!

File: client/webpack.config.js [hunk 1]
```diff
@@ -1,40 +1,34 @@
...
-const { webpack } = require("webpack");
+// const { webpack } = require("webpack");
 
 
 module.exports = {
     mode: "development",
-    entry: "/src/index.js",
-    
+
+    entry:  './src/index.js',
```
Comment (hjkmines) [inline client/webpack.config.js:9]: @FunkyColdMedyna what is this?

**Review by hjkmines — APPROVED** [review]

**Review by hjkmines — APPROVED** [review]

**Review by jkyan1988 — APPROVED** [review]
Amazing

**Comment by JosephThawn** [comment]
Looks good!

**Review by vinhpeters — APPROVED** [review]

**Pull request merged**